specutils/spectra/spectral_coordinate.py

```diff
@@ -243,7 +243,7 @@ def _copy(self, **kwargs):
     @property
     def quantity(self):
         """
-        Convert the ``SpectralCoord`` to a simple ``~astropy.units.Quantity``.
+        Convert the ``SpectralCoord`` to a simple `~astropy.units.Quantity`.
```

Comment:
```suggestion
        Convert the ``SpectralCoord`` to a `~astropy.units.Quantity`.
```
(there's a guideline somewhere in the Astropy doc dev section to avoid words like "trivial" or "simple" because they are very context-dependent)